test/cli.js

```diff
@@ -93,3 +94,57 @@ test('throwing a anonymous function will report the function to the console', fu
 		t.end();
 	});
 });
+
+test('Node.js-style --require CLI argument #1', function (t) {
+	t.plan(2);
+
+	execCli([
+		'--require=' + path.join(__dirname, 'fixture', 'install-global.js'),
+		'fixture/validate-installed-global.js'
+	], function (err, stdout, stderr) {
+		t.ifError(err);
+		t.is(stderr.trim(), '1 test passed');
+		t.end();
+	});
+});
+
+test('Node.js-style --require CLI argument #2', function (t) {
+	t.plan(2);
+
+	execCli([
+		'fixture/validate-installed-global.js',
+		'--require=' + path.join(__dirname, 'fixture', 'install-global.js')
+	], function (err, stdout, stderr) {
+		t.ifError(err);
+		t.is(stderr.trim(), '1 test passed');
+		t.end();
+	});
+});
+
+test('Node.js-style --require CLI argument #3', function (t) {
+	t.plan(2);
+
+	execCli([
+		'fixture/validate-installed-global.js',
+		'--require',
+		path.join(__dirname, 'fixture', 'install-global.js')
+	], function (err, stdout, stderr) {
+		t.ifError(err);
+		t.is(stderr.trim(), '1 test passed');
+		t.end();
+	});
+});
+
+test('Node.js-style --require CLI argument #4', function (t) {
+	t.plan(2);
+
+	execCli([
+		'--require',
+		path.join(__dirname, 'fixture', 'install-global.js'),
+		'fixture/validate-installed-global.js'
+	], function (err, stdout, stderr) {
+		t.ifError(err);
+		t.is(stderr.trim(), '1 test passed');
+		t.end();
+	});
+});
```

Comment:
Can you make these `api.js` tests instead? Spawning CLI for each test is slow and we now have an API we can test directly instead.